test/metabase/api/timeline_event_test.clj

```diff
@@ -66,3 +66,20 @@
           (is (true?
                (->> (mt/user-http-request :rasta :put 200 (str "timeline-event/" (u/the-id event)) {:archived true})
                     :archived))))))))
+
+(deftest delete-timeline-event-test
+  (testing "DELETE /api/timeline-event/:id"
+    (mt/with-temp [Collection    collection {:name "Example Data"}
+                   Timeline      timeline   {:name          "Some Events"
+                                             :collection_id (u/the-id collection)}
+                   TimelineEvent event      {:name        "Example Event"
```

Comment:
```suggestion
                   TimelineEvent {event-id :id}      {:name        "Example Event"
```

we only care about the event-id as part of the test, so might as well destructure it so we don't have to do `(u/the-id event)` later.

Reply:
same with timeline-id

Reply:
Thanks, [committed here](https://github.com/metabase/metabase/compare/76aa237563cfaeec909e00b4f8a903d9c5be51f0...f7f494ec2164fd8f006a7f752dc72279fa872238?expand=1).

…with your suggestion applied for `timeline-id` and `collection-id`.